docs/source/install.rst

```diff
@@ -88,7 +88,7 @@ These optional dependencies and their minimum supported versions are listed belo
 +===============+==========+==============================================================+
 |     bokeh     | >=1.0.0  |                Visualizing dask diagnostics                  |
 +---------------+----------+--------------------------------------------------------------+
-|  cloudpickle  | >=0.2.2  |              Pickling support for Python objects             |
+|  cloudpickle  | >=1.1.1  |              Pickling support for Python objects             |
```

Comment:
We can remove `cloudpickle` here since this is a list of optional dependencies. We can also remove `fsspec`, `partd`, and `toolz`. Though we should keep `cytoolz` as it's still optional (users don't have to use the cython implementation of `toolz`)